### PR TITLE
Update yaml schemas

### PR DESCRIPTION
**Pull request by FredZinelli**
Hello,

I updated a bunch of yaml schemas, fixing some missing configs, and adding some new elements related to projects of mine on the way.

## Fix / update:

* `schema.json` : `edit_uri_template` was missing
* `schema/pluigins`:
  I added all the `material/plugin` syntaxes to each of your plugins, because I built a new theme extending yours, [`pyodide-mkdocs-theme`](https://gitlab.com/frederic-zinelli/pyodide-mkdocs-theme) ([related question](https://github.com/squidfunk/mkdocs-material/discussions/6994)). So, I moved the object version in `$defs` and rewrote of the `"oneOf"` array, with 4 versions: 
    - `const: plugin`
    - `const: material/plugin`
    - `object:  plugin: to $defs`
    - `object:  material/plugin: to $defs`
    This fixed on the way some inconsistencies (at least one: the `group` plugin didn't have any `const` version, while it's talked about in material's docs)

## Add

* `theme.json` : added `const: pyodide-mkdocs-theme`. This one is up to you, if you don't want to add it there, no problem on my side
* `plugins.json` : this is the main reason for this PR I added 2 external refs for 2 of my projects: the plugin coming with my theme, `pyodide_macros`, and an older project of mine, `mkdocs-addresses` (the schema for this one is only a working stub, I didn't create the full config yet).

Hoping you'll be ok with the changes.

Kind regards,
Fred

---

PS: I still have one red line in my `mkdocs.yml`, but didn't find a way to fix it:

```yaml
  - pymdownx.tabbed: # Volets coulissants.  === "Mon volet"
      alternate_style: true   # compatibilité pour mobiles
      slugify: !!python/object/apply:pymdownx.slugs.slugify   # <<<< here
        kwds:
          case: lower
```

I saw the tags you used for emojis or superfences in the config of the vscode yaml extension, but I don't get what they actually are about and couldn't make something equivalent to work with the `tabbed` extension. No idea how this actually work...
It's not a problem, I'm just pointing it out, in case you know how to fix it on the way...

### CODE REVIEW

**Comment by FredZinelli** [comment]
Oh dear...! I just saw the `markdownDescription` field I added is static and always refer to the search plugin... x/
Lemme update these...

File: docs/schema/plugins/blog.json [hunk 1]
```diff
@@ -3,447 +3,467 @@
   "title": "Built-in blog plugin",
   "oneOf": [
     {
-      "markdownDescription": "https://squidfunk.github.io/mkdocs-material/plugins/blog/",
+      "markdownDescription": "https://squidfunk.github.io/mkdocs-material/plugins/search/",
```
Comment (FredZinelli) [inline docs/schema/plugins/blog.json:6]:
```suggestion
      "markdownDescription": "https://squidfunk.github.io/mkdocs-material/plugins/blog/",
```

File: docs/schema/plugins/blog.json [hunk 2]
```diff
@@ -3,447 +3,467 @@
...
       "const": "blog"
     },
+    {
+      "markdownDescription": "https://squidfunk.github.io/mkdocs-material/plugins/search/",
```
Comment (FredZinelli) [inline docs/schema/plugins/blog.json:10]:
```suggestion
      "markdownDescription": "https://squidfunk.github.io/mkdocs-material/plugins/blog/",
```

File: docs/schema/plugins/blog.json [hunk 3]
```diff
@@ -3,447 +3,467 @@
...
+      "const": "material/blog"
+    },
     {
       "type": "object",
+      "markdownDescription": "https://squidfunk.github.io/mkdocs-material/plugins/search/",
```
Comment (FredZinelli) [inline docs/schema/plugins/blog.json:15]:
```suggestion
      "markdownDescription": "https://squidfunk.github.io/mkdocs-material/plugins/blog/",
```

File: docs/schema/plugins/blog.json [hunk 4]
```diff
@@ -3,447 +3,467 @@
...
-                  "type": "string"
-                }
+          "$ref": "#/$defs/blogConfig"
+        }
+      },
+      "additionalProperties": false
+    },
+    {
+      "type": "object",
+      "markdownDescription": "https://squidfunk.github.io/mkdocs-material/plugins/search/",
```
Comment (FredZinelli) [inline docs/schema/plugins/blog.json:25]:
```suggestion
      "markdownDescription": "https://squidfunk.github.io/mkdocs-material/plugins/blog/",
```

File: docs/schema/plugins/group.json [hunk 5]
```diff
@@ -2,28 +2,52 @@
   "$schema": "https://json-schema.org/draft-07/schema",
   "title": "Built-in group plugin",
   "oneOf": [
+    {
+      "markdownDescription": "https://squidfunk.github.io/mkdocs-material/plugins/search/",
```
Comment (FredZinelli) [inline docs/schema/plugins/group.json:6]:
```suggestion
      "markdownDescription": "https://squidfunk.github.io/mkdocs-material/plugins/group/",
```

File: docs/schema/plugins/group.json [hunk 6]
```diff
@@ -2,28 +2,52 @@
...
+      "const": "group"
+    },
+    {
+      "markdownDescription": "https://squidfunk.github.io/mkdocs-material/plugins/search/",
```
Comment (FredZinelli) [inline docs/schema/plugins/group.json:10]:
```suggestion
      "markdownDescription": "https://squidfunk.github.io/mkdocs-material/plugins/group/",
```

File: docs/schema/plugins/group.json [hunk 7]
```diff
@@ -2,28 +2,52 @@
...
+      "const": "material/group"
+    },
     {
       "type": "object",
+      "markdownDescription": "https://squidfunk.github.io/mkdocs-material/plugins/search/",
```
Comment (FredZinelli) [inline docs/schema/plugins/group.json:15]:
```suggestion
      "markdownDescription": "https://squidfunk.github.io/mkdocs-material/plugins/group/",
```

File: docs/schema/plugins/group.json [hunk 8]
```diff
@@ -2,28 +2,52 @@
...
-          },
-          "additionalProperties": false
+          "$ref": "#/$defs/groupConfig"
+        }
+      },
+      "additionalProperties": false
+    },
+    {
+      "type": "object",
+      "markdownDescription": "https://squidfunk.github.io/mkdocs-material/plugins/search/",
```
Comment (FredZinelli) [inline docs/schema/plugins/group.json:25]:
```suggestion
      "markdownDescription": "https://squidfunk.github.io/mkdocs-material/plugins/group/",
```

File: docs/schema/plugins/info.json [hunk 9]
```diff
@@ -3,45 +3,65 @@
   "title": "Built-in info plugin",
   "oneOf": [
     {
-      "markdownDescription": "https://squidfunk.github.io/mkdocs-material/plugins/info/",
+      "markdownDescription": "https://squidfunk.github.io/mkdocs-material/plugins/search/",
```
Comment (FredZinelli) [inline docs/schema/plugins/info.json:6]:
```suggestion
      "markdownDescription": "https://squidfunk.github.io/mkdocs-material/plugins/info/",
```

File: docs/schema/plugins/info.json [hunk 10]
```diff
@@ -3,45 +3,65 @@
...
       "const": "info"
     },
+    {
+      "markdownDescription": "https://squidfunk.github.io/mkdocs-material/plugins/search/",
```
Comment (FredZinelli) [inline docs/schema/plugins/info.json:10]:
```suggestion
      "markdownDescription": "https://squidfunk.github.io/mkdocs-material/plugins/info/",
```

File: docs/schema/plugins/info.json [hunk 11]
```diff
@@ -3,45 +3,65 @@
...
+      "const": "material/info"
+    },
     {
       "type": "object",
+      "markdownDescription": "https://squidfunk.github.io/mkdocs-material/plugins/search/",
```
Comment (FredZinelli) [inline docs/schema/plugins/info.json:15]:
```suggestion
      "markdownDescription": "https://squidfunk.github.io/mkdocs-material/plugins/info/",
```

File: docs/schema/plugins/info.json [hunk 12]
```diff
@@ -3,45 +3,65 @@
...
-          },
-          "additionalProperties": false
+          "$ref": "#/$defs/infoConfig"
+        }
+      },
+      "additionalProperties": false
+    },
+    {
+      "type": "object",
+      "markdownDescription": "https://squidfunk.github.io/mkdocs-material/plugins/search/",
```
Comment (FredZinelli) [inline docs/schema/plugins/info.json:25]:
```suggestion
      "markdownDescription": "https://squidfunk.github.io/mkdocs-material/plugins/info/",
```

File: docs/schema/plugins/meta.json [hunk 13]
```diff
@@ -3,33 +3,53 @@
   "title": "Built-in meta plugin",
   "oneOf": [
     {
-      "markdownDescription": "https://squidfunk.github.io/mkdocs-material/plugins/meta/",
+      "markdownDescription": "https://squidfunk.github.io/mkdocs-material/plugins/search/",
```
Comment (FredZinelli) [inline docs/schema/plugins/meta.json:6]:
```suggestion
      "markdownDescription": "https://squidfunk.github.io/mkdocs-material/plugins/meta/",
```

File: docs/schema/plugins/meta.json [hunk 14]
```diff
@@ -3,33 +3,53 @@
...
       "const": "meta"
     },
+    {
+      "markdownDescription": "https://squidfunk.github.io/mkdocs-material/plugins/search/",
```
Comment (FredZinelli) [inline docs/schema/plugins/meta.json:10]:
```suggestion
      "markdownDescription": "https://squidfunk.github.io/mkdocs-material/plugins/meta/",
```

File: docs/schema/plugins/meta.json [hunk 15]
```diff
@@ -3,33 +3,53 @@
...
+      "const": "material/meta"
+    },
     {
       "type": "object",
+      "markdownDescription": "https://squidfunk.github.io/mkdocs-material/plugins/search/",
```
Comment (FredZinelli) [inline docs/schema/plugins/meta.json:15]:
```suggestion
      "markdownDescription": "https://squidfunk.github.io/mkdocs-material/plugins/meta/",
```

File: docs/schema/plugins/meta.json [hunk 16]
```diff
@@ -3,33 +3,53 @@
...
-          },
-          "additionalProperties": false
+          "$ref": "#/$defs/metaConfig"
+        }
+      },
+      "additionalProperties": false
+    },
+    {
+      "type": "object",
+      "markdownDescription": "https://squidfunk.github.io/mkdocs-material/plugins/search/",
```
Comment (FredZinelli) [inline docs/schema/plugins/meta.json:25]:
```suggestion
      "markdownDescription": "https://squidfunk.github.io/mkdocs-material/plugins/meta/",
```

File: docs/schema/plugins/offline.json [hunk 17]
```diff
@@ -3,27 +3,47 @@
   "title": "Built-in offline plugin",
   "oneOf": [
     {
-      "markdownDescription": "https://squidfunk.github.io/mkdocs-material/plugins/offline/",
+      "markdownDescription": "https://squidfunk.github.io/mkdocs-material/plugins/search/",
```
Comment (FredZinelli) [inline docs/schema/plugins/offline.json:6]:
```suggestion
      "markdownDescription": "https://squidfunk.github.io/mkdocs-material/plugins/offline/",
```

File: docs/schema/plugins/offline.json [hunk 18]
```diff
@@ -3,27 +3,47 @@
...
       "const": "offline"
     },
+    {
+      "markdownDescription": "https://squidfunk.github.io/mkdocs-material/plugins/search/",
+      "const": "material/offline"
+    },
     {
       "type": "object",
+      "markdownDescription": "https://squidfunk.github.io/mkdocs-material/plugins/search/",
```
Comment (FredZinelli) [inline docs/schema/plugins/offline.json:15]:
```suggestion
      "markdownDescription": "https://squidfunk.github.io/mkdocs-material/plugins/offline/",
```

File: docs/schema/plugins/offline.json [hunk 19]
```diff
@@ -3,27 +3,47 @@
   "title": "Built-in offline plugin",
   "oneOf": [
     {
-      "markdownDescription": "https://squidfunk.github.io/mkdocs-material/plugins/offline/",
+      "markdownDescription": "https://squidfunk.github.io/mkdocs-material/plugins/search/",
       "const": "offline"
     },
+    {
+      "markdownDescription": "https://squidfunk.github.io/mkdocs-material/plugins/search/",
```
Comment (FredZinelli) [inline docs/schema/plugins/offline.json:10]:
```suggestion
      "markdownDescription": "https://squidfunk.github.io/mkdocs-material/plugins/offline/",
```

File: docs/schema/plugins/offline.json [hunk 20]
```diff
@@ -3,27 +3,47 @@
...
-          },
-          "additionalProperties": false
+          "$ref": "#/$defs/offlineConfig"
+        }
+      },
+      "additionalProperties": false
+    },
+    {
+      "type": "object",
+      "markdownDescription": "https://squidfunk.github.io/mkdocs-material/plugins/search/",
```
Comment (FredZinelli) [inline docs/schema/plugins/offline.json:25]:
```suggestion
      "markdownDescription": "https://squidfunk.github.io/mkdocs-material/plugins/offline/",
```

File: docs/schema/plugins/optimize.json [hunk 21]
```diff
@@ -3,116 +3,136 @@
   "title": "Built-in optimize plugin",
   "oneOf": [
     {
-      "markdownDescription": "https://squidfunk.github.io/mkdocs-material/plugins/optimize/",
+      "markdownDescription": "https://squidfunk.github.io/mkdocs-material/plugins/search/",
```
Comment (FredZinelli) [inline docs/schema/plugins/optimize.json:6]:
```suggestion
      "markdownDescription": "https://squidfunk.github.io/mkdocs-material/plugins/optimize/",
```

File: docs/schema/plugins/optimize.json [hunk 22]
```diff
@@ -3,116 +3,136 @@
...
       "const": "optimize"
     },
+    {
+      "markdownDescription": "https://squidfunk.github.io/mkdocs-material/plugins/search/",
```
Comment (FredZinelli) [inline docs/schema/plugins/optimize.json:10]:
```suggestion
      "markdownDescription": "https://squidfunk.github.io/mkdocs-material/plugins/optimize/",
```

File: docs/schema/plugins/optimize.json [hunk 23]
```diff
@@ -3,116 +3,136 @@
...
+      "const": "material/optimize"
+    },
     {
       "type": "object",
+      "markdownDescription": "https://squidfunk.github.io/mkdocs-material/plugins/search/",
```
Comment (FredZinelli) [inline docs/schema/plugins/optimize.json:15]:
```suggestion
      "markdownDescription": "https://squidfunk.github.io/mkdocs-material/plugins/optimize/",
```

File: docs/schema/plugins/optimize.json [hunk 24]
```diff
@@ -3,116 +3,136 @@
...
-              "default": true
-            }
+          "$ref": "#/$defs/optimizeConfig"
+        }
+      },
+      "additionalProperties": false
+    },
+    {
+      "type": "object",
+      "markdownDescription": "https://squidfunk.github.io/mkdocs-material/plugins/search/",
```
Comment (FredZinelli) [inline docs/schema/plugins/optimize.json:25]:
```suggestion
      "markdownDescription": "https://squidfunk.github.io/mkdocs-material/plugins/optimize/",
```

File: docs/schema/plugins/privacy.json [hunk 25]
```diff
@@ -3,125 +3,145 @@
   "title": "Built-in privacy plugin",
   "oneOf": [
     {
-      "markdownDescription": "https://squidfunk.github.io/mkdocs-material/plugins/privacy/",
+      "markdownDescription": "https://squidfunk.github.io/mkdocs-material/plugins/search/",
```
Comment (FredZinelli) [inline docs/schema/plugins/privacy.json:6]:
```suggestion
      "markdownDescription": "https://squidfunk.github.io/mkdocs-material/plugins/privacy/",
```

File: docs/schema/plugins/privacy.json [hunk 26]
```diff
@@ -3,125 +3,145 @@
...
       "const": "privacy"
     },
+    {
+      "markdownDescription": "https://squidfunk.github.io/mkdocs-material/plugins/search/",
```
Comment (FredZinelli) [inline docs/schema/plugins/privacy.json:10]:
```suggestion
      "markdownDescription": "https://squidfunk.github.io/mkdocs-material/plugins/privacy/",
```

File: docs/schema/plugins/privacy.json [hunk 27]
```diff
@@ -3,125 +3,145 @@
...
+      "const": "material/privacy"
+    },
     {
       "type": "object",
+      "markdownDescription": "https://squidfunk.github.io/mkdocs-material/plugins/search/",
```
Comment (FredZinelli) [inline docs/schema/plugins/privacy.json:15]:
```suggestion
      "markdownDescription": "https://squidfunk.github.io/mkdocs-material/plugins/privacy/",
```

File: docs/schema/plugins/privacy.json [hunk 28]
```diff
@@ -3,125 +3,145 @@
...
         "privacy": {
-          "markdownDescription": "https://squidfunk.github.io/mkdocs-material/plugins/privacy/",
+          "$ref": "#/$defs/privacyConfig"
+        }
+      },
+      "additionalProperties": false
+    },
+    {
+      "type": "object",
+      "markdownDescription": "https://squidfunk.github.io/mkdocs-material/plugins/search/",
```
Comment (FredZinelli) [inline docs/schema/plugins/privacy.json:25]:
```suggestion
      "markdownDescription": "https://squidfunk.github.io/mkdocs-material/plugins/privacy/",
```

File: docs/schema/plugins/projects.json [hunk 29]
```diff
@@ -3,95 +3,115 @@
   "title": "Built-in projects plugin",
   "oneOf": [
     {
-      "markdownDescription": "https://squidfunk.github.io/mkdocs-material/plugins/projects/",
+      "markdownDescription": "https://squidfunk.github.io/mkdocs-material/plugins/search/",
```
Comment (FredZinelli) [inline docs/schema/plugins/projects.json:6]:
```suggestion
      "markdownDescription": "https://squidfunk.github.io/mkdocs-material/plugins/projects/",
```

File: docs/schema/plugins/projects.json [hunk 30]
```diff
@@ -3,95 +3,115 @@
...
       "const": "projects"
     },
     {
+      "markdownDescription": "https://squidfunk.github.io/mkdocs-material/plugins/search/",
```
Comment (FredZinelli) [inline docs/schema/plugins/projects.json:10]:
```suggestion
      "markdownDescription": "https://squidfunk.github.io/mkdocs-material/plugins/projects/",
```

File: docs/schema/plugins/projects.json [hunk 31]
```diff
@@ -3,95 +3,115 @@
...
+      "const": "material/projects"
+    },
+    {
+      "type": "object",
+      "markdownDescription": "https://squidfunk.github.io/mkdocs-material/plugins/search/",
```
Comment (FredZinelli) [inline docs/schema/plugins/projects.json:15]:
```suggestion
      "markdownDescription": "https://squidfunk.github.io/mkdocs-material/plugins/projects/",
```

File: docs/schema/plugins/projects.json [hunk 32]
```diff
@@ -3,95 +3,115 @@
...
+      "properties": {
+        "projects": {
+          "$ref": "#/$defs/projectsConfig"
+        }
+      },
+      "additionalProperties": false
+    },
+    {
+      "type": "object",
+      "markdownDescription": "https://squidfunk.github.io/mkdocs-material/plugins/search/",
```
Comment (FredZinelli) [inline docs/schema/plugins/projects.json:25]:
```suggestion
      "markdownDescription": "https://squidfunk.github.io/mkdocs-material/plugins/projects/",
```

File: docs/schema/plugins/social.json [hunk 33]
```diff
@@ -3,155 +3,175 @@
   "title": "Built-in social plugin",
   "oneOf": [
     {
-      "markdownDescription": "https://squidfunk.github.io/mkdocs-material/plugins/social/",
+      "markdownDescription": "https://squidfunk.github.io/mkdocs-material/plugins/search/",
```
Comment (FredZinelli) [inline docs/schema/plugins/social.json:6]:
```suggestion
      "markdownDescription": "https://squidfunk.github.io/mkdocs-material/plugins/social/",
```

File: docs/schema/plugins/social.json [hunk 34]
```diff
@@ -3,155 +3,175 @@
...
       "const": "social"
     },
+    {
+      "markdownDescription": "https://squidfunk.github.io/mkdocs-material/plugins/search/",
```
Comment (FredZinelli) [inline docs/schema/plugins/social.json:10]:
```suggestion
      "markdownDescription": "https://squidfunk.github.io/mkdocs-material/plugins/social/",
```

File: docs/schema/plugins/social.json [hunk 35]
```diff
@@ -3,155 +3,175 @@
...
+      "const": "material/social"
+    },
     {
       "type": "object",
+      "markdownDescription": "https://squidfunk.github.io/mkdocs-material/plugins/search/",
```
Comment (FredZinelli) [inline docs/schema/plugins/social.json:15]:
```suggestion
      "markdownDescription": "https://squidfunk.github.io/mkdocs-material/plugins/social/",
```

File: docs/schema/plugins/social.json [hunk 36]
```diff
@@ -3,155 +3,175 @@
...
-              "title": "Log level",
-              "markdownDescription": "https://squidfunk.github.io/mkdocs-material/plugins/social/#config.log_level",
+          "$ref": "#/$defs/socialConfig"
+        }
+      },
+      "additionalProperties": false
+    },
+    {
+      "type": "object",
+      "markdownDescription": "https://squidfunk.github.io/mkdocs-material/plugins/search/",
```
Comment (FredZinelli) [inline docs/schema/plugins/social.json:25]:
```suggestion
      "markdownDescription": "https://squidfunk.github.io/mkdocs-material/plugins/social/",
```

File: docs/schema/plugins/tags.json [hunk 37]
```diff
@@ -3,223 +3,243 @@
   "title": "Built-in tags plugin",
   "oneOf": [
     {
-      "markdownDescription": "https://squidfunk.github.io/mkdocs-material/plugins/tags/",
+      "markdownDescription": "https://squidfunk.github.io/mkdocs-material/plugins/search/",
```
Comment (FredZinelli) [inline docs/schema/plugins/tags.json:6]:
```suggestion
      "markdownDescription": "https://squidfunk.github.io/mkdocs-material/plugins/tags/",
```

File: docs/schema/plugins/tags.json [hunk 38]
```diff
@@ -3,223 +3,243 @@
...
       "const": "tags"
     },
     {
+      "markdownDescription": "https://squidfunk.github.io/mkdocs-material/plugins/search/",
```
Comment (FredZinelli) [inline docs/schema/plugins/tags.json:10]:
```suggestion
      "markdownDescription": "https://squidfunk.github.io/mkdocs-material/plugins/tags/",
```

File: docs/schema/plugins/tags.json [hunk 39]
```diff
@@ -3,223 +3,243 @@
...
+      "const": "material/tags"
+    },
+    {
+      "type": "object",
+      "markdownDescription": "https://squidfunk.github.io/mkdocs-material/plugins/search/",
```
Comment (FredZinelli) [inline docs/schema/plugins/tags.json:15]:
```suggestion
      "markdownDescription": "https://squidfunk.github.io/mkdocs-material/plugins/tags/",
```

File: docs/schema/plugins/tags.json [hunk 40]
```diff
@@ -3,223 +3,243 @@
...
+      "properties": {
+        "tags": {
+          "$ref": "#/$defs/tagsConfig"
+        }
+      },
+      "additionalProperties": false
+    },
+    {
+      "type": "object",
+      "markdownDescription": "https://squidfunk.github.io/mkdocs-material/plugins/search/",
```
Comment (FredZinelli) [inline docs/schema/plugins/tags.json:25]:
```suggestion
      "markdownDescription": "https://squidfunk.github.io/mkdocs-material/plugins/tags/",
```

File: docs/schema/plugins/typeset.json [hunk 41]
```diff
@@ -3,27 +3,47 @@
   "title": "Built-in typeset plugin",
   "oneOf": [
     {
-      "markdownDescription": "https://squidfunk.github.io/mkdocs-material/plugins/typeset/",
+      "markdownDescription": "https://squidfunk.github.io/mkdocs-material/plugins/search/",
```
Comment (FredZinelli) [inline docs/schema/plugins/typeset.json:6]:
```suggestion
      "markdownDescription": "https://squidfunk.github.io/mkdocs-material/plugins/typeset/",
```

File: docs/schema/plugins/typeset.json [hunk 42]
```diff
@@ -3,27 +3,47 @@
...
       "const": "typeset"
     },
+    {
+      "markdownDescription": "https://squidfunk.github.io/mkdocs-material/plugins/search/",
```
Comment (FredZinelli) [inline docs/schema/plugins/typeset.json:10]:
```suggestion
      "markdownDescription": "https://squidfunk.github.io/mkdocs-material/plugins/typeset/",
```

File: docs/schema/plugins/typeset.json [hunk 43]
```diff
@@ -3,27 +3,47 @@
...
+      "const": "material/typeset"
+    },
     {
       "type": "object",
+      "markdownDescription": "https://squidfunk.github.io/mkdocs-material/plugins/search/",
```
Comment (FredZinelli) [inline docs/schema/plugins/typeset.json:15]:
```suggestion
      "markdownDescription": "https://squidfunk.github.io/mkdocs-material/plugins/typeset/",
```

File: docs/schema/plugins/typeset.json [hunk 44]
```diff
@@ -3,27 +3,47 @@
...
-          },
-          "additionalProperties": false
+          "$ref": "#/$defs/typesetConfig"
+        }
+      },
+      "additionalProperties": false
+    },
+    {
+      "type": "object",
+      "markdownDescription": "https://squidfunk.github.io/mkdocs-material/plugins/search/",
```
Comment (FredZinelli) [inline docs/schema/plugins/typeset.json:25]:
```suggestion
      "markdownDescription": "https://squidfunk.github.io/mkdocs-material/plugins/typeset/",
```

**Comment by FredZinelli** [comment]
(Sorry for the noisy commits, I'm not used to github management/PR... 'x) )

**Comment by squidfunk** [comment]
Thanks for the PR and for all your work. However, 1,444 lines changed – it's impossible for us to review this. There are definitely some things that make no sense to merge from our perspective, for example the group plugin cannot have a non-object syntax, because it just makes no sense. It would be a no-op. Additionally, combining `material/x` and `x` into one should be possible with patterns. We can't just merge a change this large, because it will impact all of our users that use the schema as an aid in their IDEs, and dissecting and discussing it will take too much effort.

Please cut this PR into several smaller PRs, e.g., those that just add missing MkDocs settings, and those that do fundamental changes, so they can be separately discussed and tested. You do not need to make a PR for each change introduced, but group them logically, so they are easy to reason about. Please try to make those PRs self-contained and easy to reason about, then we can think about merging them.

Closing, as this is not possible to merge.

**Comment by FredZinelli** [comment]
Ok, no problem. I'll get back to you with something more concise.

**Pull request closed without merging**